src/main/play-25/twirl/views/layouts/govuk_template.scala.html

```diff
@@ -15,23 +15,30 @@
  *@
 
 @(title: Option[String],
-        bodyClasses: Option[String])(
-        head: Html,
-        bodyEnd: Html,
-        insideHeader: Html,
-        afterHeader: Html,
-        footerTop: Html,
-        footerLinks: Option[Html],
-        nav: Boolean = false)(
-        content: Html)
+  bodyClasses: Option[String])(
+  head: Html,
+  bodyEnd: Html,
+  insideHeader: Html,
+  afterHeader: Html,
+  footerTop: Html,
+  footerLinks: Option[Html],
+  nav: Boolean = false)(
+  content: Html
+)(implicit messages: Messages)
+
 @import template._
+
+@lang = @{
+  messages.lang.code
+}
+
 <!DOCTYPE html>
-<!--[if lt IE 9]><html class="no-js lte-ie8" lang="en"><![endif]-->
-<!--[if gt IE 8]><!--><html lang="en" class="no-js"><!--<![endif]-->
+<!--[if lt IE 9]><html class="no-js lte-ie8" lang=@lang><![endif]-->
+<!--[if gt IE 8]><!--><html lang=@lang class="no-js"><!--<![endif]-->
     <head>
         <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
-        <title>@title.getOrElse("GOV.UK - The best place to find government services and information")</title>
+        <title>@title.getOrElse(Messages("base.title.default"))</title>
```

Comment:
You have brought implicit messages in however use the Messages object. Please switch this for all messages.